blaze/expr/strings.py

```diff
@@ -74,6 +84,54 @@ def schema(self):
         return self._child.schema
 
 
+class StrCat(ElemWise):
+    """
+    Concatenate two string columns together with optional 'sep' argument.
+
+    >>> import pandas as pd
+    >>> from blaze import symbol, compute, dshape
+
+    >>> ds = dshape('3 * {name: string[10], comment: string[25], num: int32}')
+    >>> s = symbol('s', dshape=ds)
+    >>> data = [('al', 'good', 0), ('suri', 'not good', 1), ('jinka', 'ok', 2)]
```

Comment:
Can you add a row with `None` in there for one of the strings, to show what happens when a string is missing, and that `str_cat()` propagates `None` in this case?